From bead66431b1dbc17fdb201bf5e7d3db86e1f49a7 Mon Sep 17 00:00:00 2001
From: AndroPlus <mail@andro.plus>
Date: Sat, 23 Mar 2019 13:39:55 +0900
Subject: [PATCH 1/1] Revert "SystemUI: Fix an issue where jumping to black
 after screen turned on"

This reverts commit 01c063e0f168bc40e74acc6390546cef1d0ea20e.

Signed-off-by: AndroPlus <mail@andro.plus>
---
 .../src/com/android/systemui/statusbar/phone/StatusBar.java    | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java b/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java
index 22fa5197789..9ddefb2e55c 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/phone/StatusBar.java
@@ -5438,7 +5438,7 @@ public class StatusBar extends SystemUI implements DemoMode, TunerService.Tunabl
 
         @Override
         public void startDozing() {
-            if (!mDozingRequested && !isScreenTurningOnOrOn()) {
+            if (!mDozingRequested) {
                 mDozingRequested = true;
                 DozeLog.traceDozing(mContext, mDozing);
                 updateDozing();
@@ -5503,7 +5503,6 @@ public class StatusBar extends SystemUI implements DemoMode, TunerService.Tunabl
                 DozeLog.traceDozing(mContext, mDozing);
                 mWakefulnessLifecycle.dispatchStartedWakingUp();
                 updateDozing();
-                updateScrimController();
 
                 if (isAmbientContainerAvailable()) {
                     ((AmbientIndicationContainer)mAmbientIndicationContainer).setDozing(false);
-- 
2.17.1

